### DISCUSSION TITLE
Gesamtertrag nur 75% vom gemessenen Wert (HM-800)

### DISCUSSION THREAD

**Discussion by JoeJCool** [tbnobody/OpenDTU#1384] [Q&A]
Hallo zusammen,

seit einer Woche spiele ich mit openDTU. Ein schönes Projekt, das macht Spass :)

Den HM-800 (zwei Module) hatte ich einmal temporär auf 600W begrenzt, so dass -wie ich den anderen Diskussionen entnommen habe- nach der automatischen Nachtabschaltung am nächsten Tag das Limit wieder bei 800W liegen sollte. Einmal hatte ich bei praller Sonne die 800W laut openDTU erreicht.

Zum Vergleich der Leistung/Erträge lasse ich seit zwei Tagen eine kalibrierte Tasmota-Steckdose (Nous A1T) mitlaufen. Die Werte der Gesamtleistung liegen nahe beieinander, openDTU gibt etwa 12W weniger an. Der Gesamtertrag des Tages unterscheidet sich aber um etwa 25% (openDTU 2430 vs. Tasmota 3133Wh), was ich mir durch die 12W nicht erklären kann. Da die 25% der Begrenzung vom 800 auf 600W entsprechen, habe ich vermutet, das der Wechselrichter bei der Berechnung irgendwo einen Fehler macht (was aber dann auch in der Gesamtleistung zu sehen sein sollte). Einmalig habe ich das Limit gestern manuell auf 800W gesetzt (Wechselrichter und openDTU heute morgen neu gestartet) und beobachte die gleiche Abweichung.

Auf die doppelte Erfassung würde ich zukünftig verzichten wollen und gerne als nur die Werte aus openDTU nehmen (MQTT in Planung). Entweder ich korrigiere manuell mit einem empirischen Faktor den (vermutlich) falschen Wert aus dem Wechselrichter oder besser jemand hat eine Idee, wie die Werte direkt richtig ausgegeben werden. :)

Vielen Dank und sonnige Grüße, Joe

**Comment by DPO99** [comment]
Also ich nutz einen ShellyPlus2PM dazwischen und habe eine Differenz von ca. 8%. Also Shelly zeigt 8% weniger an als der WR. Unterm Strich muss man aber sagen, wenn du korrekt messen willst, dann musst du auch korrekt messen. So eine halbwegs schlecht und weniger recht kalibrierte Steckdose ist da nicht hilfreich. Und wer weiß wie genau der WR misst. Also ich würde dir ein wirkliches Messerkzeug (Shelly EM z.B.) empfehlen, wenn du das ganz genau haben willst. Alternativ gehen ja auch geeichte Stromzähler von Eltako z.B..

**Reply by JoeJCool** [reply]
Danke! Daran hab ich natürlich noch nicht gedacht, die Steckdose zu überprüfen.... (Vertrauen ist gut, Kontrolle der Kontrolle ist besser :))
